Food.c

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#define MAX_FOOD_NAME 20
+//
+// Created by razvi on 11/5/2019.
+//
+void ChooseFood(int *state, int *nrMenu, int *FoodChoice){
```

Comment:
you have function names starting with a lowercase letter, I would stick with that for C